test/testmetrics/metrics-hpa/01-assert.yaml

```diff
@@ -13,3 +13,14 @@ spec:
     metadata:
       labels:
         component: podtato-head-entry
+---
+apiVersion: kuttl.dev/v1beta1
```

Comment:
@rizul2108 you cannot have a deploymnet assert and a test assert defined in the same file, you should move the deployment in a separate yaml file and call it inside an assert in the TestAssert similarly to what is done in the other test you modified

Reply:
Ok, on it

Reply:
done @RealAnna